README.md

```diff
@@ -153,6 +155,8 @@ As a starting point, we recommend [EventStorming](https://www.eventstorming.com/
 
 ![EventStorming](resources/event_storming.jpeg)
 
+More details regarding EventStorming are available in [this repository](https://github.com/ddd-crew/eventstorming-glossary-cheat-sheet)
```

Comment:
There's a link to EventStorming right before the image and one 4 lines below, it feels like adding another link here is redundant.

If the goal is to add a link to the DDD Crew EventStorming repo I would change one/both of the existing links.

Reply:
```suggestion
```